avocado/utils/output.py

```diff
@@ -44,6 +45,28 @@ def display_data_size(size):
     return '%.2f %s' % (size, prefixes[i])
 
 
+def convert_data_size(size, default_sufix='B'):
+    '''
+    Convert data size from human readable units to an int of arbitrary size.
+    :param size: Human readable data size representation (string).
+    :param default_sufix: Default sufix used to represent data.
+    :return: Int with data size in the appropriate order of magnitude.
```

Comment:
By `appropriate order of magnitude` you mean `bytes`? Why not just say `in bytes`?